### PR TITLE
da-DK: Update scenario descriptions and apply #2176, #2181, #2188, #2199, #2205, #2214, #2220

### PR DESCRIPTION
**Pull request by LPSGizmo**
Fixes some typos, finalizing Scenarios descriptions.

100% DONE

and
Applying for issue(s):
- #2176 
- #2181 
- #2188 
- #2199 
- #2205 
- #2214 
- #2220

### CODE REVIEW

File: data/language/da-DK.txt [hunk 1]
```diff
@@ -4201,243 +4220,243 @@ STR_DTLS    :Du starter med en lille markeds bazaar, din udfordring er at forøg
...
 STR_SCNR    :Six Flags over Texas
 STR_PARK    :Six Flags over Texas
-STR_DTLS    :Build the missing Six Flags forlystelser, or create your own designs to improve the park! But don't forget your ultimate aim - To attract more guests to the park!
+STR_DTLS    :Byg de manglende Six Flags forlystelser, eller opret dine egne designs for at forbedre parken! Men glem ikke dit ultimative mål - Tiltrække flere gæster til parken!
 
 ###############################################################################
 ## Wacky Worlds Scenarios
 ###############################################################################
 <Africa - African Diamond Mine>
 STR_SCNR    :Africa - African Diamond Mine
```
Comment (Gymnasiast) [inline data/language/da-DK.txt:4304]: For the WW and TT scenarios, it is intended to also translate STR_SCNR.

Comment (LPSGizmo) [reply data/language/da-DK.txt:4304]: Will do, and this does only apply to Wacky Worlds and Time Twister Scenarios?

Comment (Gymnasiast) [reply data/language/da-DK.txt:4304]: Yup, and then only to STR_SCNR, not STR_PARK.

Comment (LPSGizmo) [reply data/language/da-DK.txt]: Done (again) :)

**Pull request merged**